boards/nrf52840dongle/Makefile.dep

```diff
@@ -0,0 +1,11 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+# ethos only works on the UART-based stdio; when present, don't attempt to use CDC-ACM...
```

Comment:
```suggestion
# use nrfutil-bootloader/stdio_cdc_acm only if no other stdio_% other than stdio_cdc_acm
# is requested
```

Reply:
I don't see how stdio_cdc_acm is related to using the nrfutil bootloader. I can use USB to flash a device like this but still use stdio_uart as the user facing mechanism -- or (if/when riotboot support is added so it can be used to replace nrfutil) use stdio_cdc_acm with riotboot. The workaround here so far is only to make mechanisms like ethos that fail with any non-uart stdio fall back to that stdio (and that, too, is independent of nrfutil). But maybe I get the intended change wrong...

Reply:
Ok ,you can remove the mention to nrfutil-bootloader if you prefer but the rest is still valid and better than the current comment (it's not only about ethos)